### PR TITLE
Add detail to website property

### PR DESCRIPTION
**Pull request by noon-dawg**
## Summary

This pull request add detail to the website property.

## Checklist

- [X] Writing style
- [X] API design standards

### CODE REVIEW

File: openapi/components/schemas/Organizations/Organization.yaml [hunk 1]
```diff
@@ -18,8 +18,12 @@ properties:
     maxLength: 60
     x-sortable: true
   website:
-    description: Organization's website.
+    description: |-
+      Website URL of the organization.
+      This field accepts unicode characters,
```
Comment (nanobash) [inline openapi/components/schemas/Organizations/Organization.yaml:23]: I am not sure this line is appropriate as in website URL there should not be unicode chars.

```suggestion
```

File: openapi/components/schemas/Organizations/Organization.yaml [hunk 2]
```diff
@@ -18,8 +18,12 @@ properties:
     maxLength: 60
     x-sortable: true
   website:
-    description: Organization's website.
+    description: |-
+      Website URL of the organization.
+      This field accepts unicode characters, 
+      and the value must begin with `https://` or `http://`.
```
Comment (ashkarpetin) [inline openapi/components/schemas/Organizations/Organization.yaml:24]: We can add example.

File: openapi/components/schemas/Organizations/Organization.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (noon-dawg) [reply openapi/components/schemas/Organizations/Organization.yaml:23]: Good point. Thank you 🙏

File: openapi/components/schemas/Organizations/Organization.yaml [hunk 3]
```diff
@@ -18,8 +18,12 @@ properties:
...
     type: string
+    maxLength: 65
     writeOnly: true
     format: url
```
[the remark this answers is not in the document]
Comment (noon-dawg) [reply openapi/components/schemas/Organizations/Organization.yaml:28]: Thank you!

File: openapi/components/schemas/Organizations/Organization.yaml [hunk 4]
```diff
@@ -18,10 +18,14 @@ properties:
...
   website:
-    description: Organization's website.
+    description: |-
+      Website URL of the organization.
+      This value must begin with `https://` or `http://`.
     type: string
+    maxLength: 65
     writeOnly: true
     format: url
+    example: https//example.com
```
[the remark this answers is not in the document]
Comment (noon-dawg) [reply openapi/components/schemas/Organizations/Organization.yaml:28]: Well spotted! Thank you 😅

**Review by nanobash — APPROVED** [review]

**Review by ashkarpetin — APPROVED** [review]

**Pull request merged**